### PR TITLE
cookbook - multiclass lda

### PR DESCRIPTION
**Pull request by OXPHOS**

### CODE REVIEW

File: doc/cookbook/source/examples/multiclass_classifier/multiclass_lda.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+========================================
+Multi-class Linear Discriminant Analysis
+========================================
```
Comment (karlnapf) [inline doc/cookbook/source/examples/multiclass_classifier/multiclass_lda.rst]: no need for multiclass in name

Comment (karlnapf) [inline doc/cookbook/source/examples/multiclass_classifier/multiclass_lda.rst]: not title

File: doc/cookbook/source/examples/multiclass_classifier/multiclass_lda.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The application of
```
Comment (karlnapf) [inline doc/cookbook/source/examples/multiclass_classifier/multiclass_lda.rst]: This is no sentence :)

**Comment by karlnapf** [comment]
@vigsterkr why is there no cookbook preview here?

**Comment by vigsterkr** [comment]
there is!

**Comment by vigsterkr** [comment]
http://buildbot.shogun-toolbox.org/cookbook_pr/9a0a43e470ea8cc38e849d38c62c5dc3494ec510/

**Comment by karlnapf** [comment]
Almost ready to go

**Comment by karlnapf** [comment]
weird that my github doesnt show it

**Comment by vigsterkr** [comment]
click on `show all checks`

**Comment by karlnapf** [comment]
@OXPHOS can you send a separate patch that removes "multiclass" from all titles and filenames?

**Comment by karlnapf** [comment]
I did, there is only travis ci here

**Comment by vigsterkr** [comment]
for me only `default` is there = PR buildbot

**Pull request closed without merging**